dataland-framework-toolbox/data-dictionary/README.MD

```diff
@@ -0,0 +1,11 @@
+# Data Dictionary XLSX conversion
```

Comment:
Darüber sollten wir mal sprechen.

Reply:
Split xlsx file and integrate into process

Reply:
Done. SFDR now directly parses and converts XLSX on every toolbox run. (Tested on Windows & Linux)